games/a.yaml

```diff
@@ -51,6 +51,31 @@
   updated: 2016-12-11
   url: https://achtungkurve.com/
 
+- name: Achtung, die Kurve!
+  images:
+  - https://kurve.se/screenshots/kurve-round.png
+  - https://kurve.se/screenshots/kurve-splash.png
+  - https://kurve.se/screenshots/kurve-lobby.png
+  info: The original <kbd>ZATACKA.EXE</kbd> in the browser
+  framework:
+  - HTML
+  - CSS
+  lang:
+  - JavaScript
+  license:
+  - AGPL3
+  development: complete
+  originals:
+  - Achtung, die Kurve!
+  repo: https://github.com/SimonAlling/kurve
+  content: open
+  status: playable
+  type: clone
+  multiplayer:
+  - Split-screen
```

Comment:
hmm our schema breaks down here; it's shared screen competitive gameplay. Not sure what we should do here

Reply:
Me neither. I tried to pick the best one out of the available options. I don’t know exactly what to do, but I know one thing for sure: It must say multiplayer one way or another.

Reply:
Well since we're using giantbomb for genres, I checked and here's their multiplayer features list:

<select id="multiplayer" name="multiplayer[]" multiple="multiple"><option value="25">Asynchronous multiplayer</option><option value="16">LAN co-op</option><option value="19">LAN competitive</option><option value="15">Local co-op</option><option value="18">Local competitive</option><option value="21">Local splitscreen</option><option value="17">Online co-op</option><option value="20">Online competitive</option><option value="22">Online splitscreen</option><option value="23">Pass and play</option><option value="24">Voice chat</option></select>

So I guess we can add "Competitive" to our schema